public/editor-client/src/types/DefaultTemplate.ts

```diff
@@ -280,3 +271,60 @@ export interface LayoutsPagesResult {
   styles: Style;
 }
 // endregion
+
+// region Story Types
+export interface StoriesAPI {
+  title: string;
+  categories: string;
+  id: string;
+  pages: number;
+  thumbnail: string;
+  thumbnailWidth: number;
+  thumbnailHeight: number;
+}
+
+export interface StoryPages {
+  slug: string;
+  thumbnailHeight: number;
+  thumbnailWidth: number;
+  thumbnail: string;
+}
```

Comment:
create common type with repated keys and use it as extend for `StoriesAPI` and `StoryPages`